crates/bevy_math/src/face_toward.rs

```diff
@@ -1,8 +1,51 @@
 use crate::{Mat4, Vec3};
 
-/// Generates a translation / rotation matrix that faces a given target
+/// Implements a function to generate a translation / rotation matrix that faces a given target.
```

Comment:
```suggestion
/// Implements a function that generates a translation / rotation matrix that faces a given target.
```

I think this reads slightly more clearly.

Reply:
Done.